inf-clojure.el

```diff
@@ -577,7 +577,7 @@ The prefix argument SWITCH-TO-REPL controls whether to switch to REPL after the
 If you are using REPL types, it will pickup the most approapriate
 `inf-clojure-var-doc-form` variant."
   (pcase inf-clojure-repl-type
-    (lumo inf-clojure-var-doc-form-lumo)
```

Comment:
I'm 100% that this is the correct syntax for `pcase` symbols and it's generally better to use symbols over strings (strings are not interned in Elisp). If cider uses only strings in such cases it's just accidental.